demo/WebexMeetingsWidgetDemo.jsx

```diff
@@ -82,6 +104,11 @@ export default function WebexMeetingsWidgetDemo({token, fedramp}) {
         <Button disabled={!displayWidget} onClick={handleFullscreen} ariaLabel="Display meeting widget full screen">
           Fullscreen
         </Button>
+        <div className="webex-select-control">
+          <div id="display-meeting-widget-status" className="webex-error">
+            {spaceIDError && spaceIDError}
+          </div>
+        </div>
```

Comment:
```
{
spaceIDError &&
<div className="webex-select-control">
  <div id="display-meeting-widget-status" className="webex-error">
    {spaceIDError}
  </div>
</div>
}
```

Reply:
I mean both are all the same, the only thing is `meeting-widget-status` will not present in `DOM` until spaceIDError, will update..